src/main.service.js

```diff
@@ -160,6 +160,28 @@ plugin.service('wgnConfigSrv', ['$q', '$firebase', 'znData', function ($q, $fire
 		return multi ? saveMulti(workspaceId, $ref, config) : saveSingle(workspaceId, $ref);
 	};
 
+	/**
+	 * Saves a plugin's secure configuration settings for the given workspace.
+	 *
+	 * @param {number} workspaceId
+	 * @param {String} configId The config id for multi configurations
+	 * @param {Object} config The configuration object to save.
+	 *
+	 * @return {Promise<*>}
+	 */
+	srv.saveSecure = function (workspaceId, configId, config) {
+
+		var params = {
+			workspaceId: workspaceId,
+			params: {
+				configId: configId
```

Comment:
I think the above comment addresses this as well. I originally did have configId in a conditional, but simplified it since it can't be used that way.

Let me know if you were thinking something else.

Reply:
nah, that's great. Thanks, Wes!